google-common/src/main/scala/akka/stream/alpakka/google/http/GoogleHttp.scala

```diff
@@ -35,9 +35,9 @@ private[alpakka] object GoogleHttp {
 @InternalApi
 private[alpakka] final class GoogleHttp private (val http: HttpExt) extends AnyVal {
 
-  private implicit def system = http.system
-  private implicit def ec = system.dispatcher
-  private implicit def scheduler = system.scheduler
+  private implicit def system: ExtendedActorSystem = http.system
```

Comment:
Are you sure this needs to be extended actorsystem, just ActorSystem is probably enough.